PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md

```diff
@@ -122,3 +110,20 @@ echo "Installed Microsoft Office"
 rm -r /tmp/$TempFolder
 
 echo "Deleted /tmp/$TempFolder"
+```
+
+#### Description
+
+This command pulls the latest Microsoft Office 365 installer .pkg from the URL specified in the DownloadURL variable. This is designed to install Microsoft Office for Mac for Office 365 users. Please note, at the time of this example posting, linkid: 525133 was the latest version available, this may change in the future.
+
+It's advised to set a longer timeout window for this command, at the time of writing the Office 365 Installer was 1.5GB. Download and install time must be taken into consideration. In testing, a timeout of 800 seconds proved adequate.
+
+This is a community written command from [joffems](https://github.com/joffems) thanks for contributing!
+
+#### *Import This Command*
+
+To import this command into your JumpCloud tenant run the below command using the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Installing-the-JumpCloud-PowerShell-Module)
+
+```
+Import-JCCommand -URL 'https://REPLACEME'
```

Comment:
Do we need to replace "REPLACEME" with:
Mac - Install Microsoft Office 365 | v1.0 JCCG

Reply:
Long story short, we replace the link *after* is published to master with a shortened URL